src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,5 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white: #fff;
+$card-width: 200px;
```

Comment:
Don't use name of the color in the variable name. Also add `-color` at the end of the variable name
For example `$secondary-color`